grails-app/views/occurrence/_facets.gsp

```diff
@@ -86,25 +83,49 @@ ${alatag.logMsg(msg:"Start of facets.gsp - " + startTime)}
                             <thead class="fixedHeader">
                                 <tr class="tableHead">
-                                    <th>&nbsp;</th>
-                                    <th id="indexCol" width="80%"><a href="#index" class="fsort" data-sort="index" data-foffset="0"></a></th>
+                                    <th>
+                                        &nbsp;
+                                    </th>
+                                    <th id="indexCol" width="80%">
+                                        <a
+                                            href="#index"
+                                            class="fsort"
+                                            data-sort="index"
+                                            data-foffset="0"
+                                            title="<g:message code='list.table.sortBy.label' />"
+                                        </a>
+                                    </th>
                                     <th style="border-right-style: none;text-align: right;">
-                                        <a href="#count" class="fsort" data-sort="count" data-foffset="0" title="Sort by record count">
-                                            <g:message code="facets.multiplefacets.tableth01" default="Count"/>
+                                        <a
+                                            href="#count"
+                                            class="fsort"
+                                            data-sort="count"
+                                            data-foffset="0"
+                                            title="<g:message code='facets.multiplefacets.tableth01.sort' />"
+                                        >
+                                            <g:message code="facets.multiplefacets.tableth01"/>
                                         </a>
                                     </th>
                                 </tr>
                             </thead>
 
                             <tbody class="scrollContent">
-                                %{-- XXX Hide doesn't work with Bootstrap 4.--}%
-                                <tr class="hide">
-                                    <td><input type="checkbox" name="fqs" class="fqs" value=""></td>
-                                    <td><a href=""></a></td>
-                                    <td style="text-align: right; border-right-style: none;"></td>
+                                %{-- What is this hiden row for? Though seems like a hack - somewhere it is popped or is it? --}%
+                                <tr style="display: none;">
+                                    <td>
+                                        <input type="checkbox" name="fqs" class="fqs" value="">
+                                    </td>
+                                    <td>
+                                        <a href=""></a>
```

Comment:
don't know, some placeholder for js magic, have currently other stuff and this works